tasks/publish-to-mrrc/publish-to-mrrc.yaml

```diff
@@ -0,0 +1,75 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: publish-to-mrrc
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  params:
+    - name: mrrcParamFilePath
+      description: path of the env file for mrrc parameters to use
+      type: string
+    - name: cosignPubKeyConfig
+      description: the configmap name for cosign verify public key
+      type: string
+    - name: charonAWSSecret
+      description: the secret name for charon aws credential file
+      type: string
+  steps:
+    - name: verify-repo-signature
```

Comment:
If all the steps end up using the same image in the end (the release-service-utils image), maybe it can all be done in one step? It's ok to have more steps if it helps with readability or debugging of issues. But here it seems that it's pretty small overall (once the duplicate things are removed) and maybe having multiple steps does not have much benefit and it's just extra overhead to spin up two extra containers. WDYT?

Reply:
If the images can be consolidated together, I agree to use single step to do this which makes it more readable. But I'm really not sure if the charon image is easy to be integrated into release-service-utils. The reason is discussed in following thread.

Reply:
@mmalina I think we agreed to keep the charon image separated, so is this thread ok to resolve?

Reply:
Yep, thanks.